docs/source/extended.md

```diff
@@ -1,10 +1,10 @@
 
 # Translation
 
-This example is for training for the WMT'16 Multimodal Translation task (http://www.statmt.org/wmt16/multimodal-task.html). It will use on the fly tokenization with [pyonmttok](https://github.com/OpenNMT/Tokenizer) and [sacrebleu](https://github.com/mjpost/sacrebleu) for evaluation.
+This example is for training for the WMT'14 English to German news translation task (https://www.statmt.org/wmt14/translation-task.html). It will use on the fly tokenization with [sentencepiece](https://github.com/google/sentencepiece) and [sacrebleu](https://github.com/mjpost/sacrebleu) for evaluation.
```

Comment:
Can be better?
WMT'14 English to German news translation task (https://www.statmt.org/wmt14/translation-task.html) -> [WMT'14 English to German news translation task](https://www.statmt.org/wmt14/translation-task.html)

Reply:
Indeed, forgot the brackets.